nemo/core/classes/modelPT.py

```diff
@@ -120,10 +122,21 @@ def restore_from(cls, restore_path: str):
 
         with tempfile.TemporaryDirectory() as tmpdir:
             cls.__unpack_nemo_file(path2file=restore_path, out_folder=tmpdir)
-            config_yaml = path.join(tmpdir, _MODEL_CONFIG_YAML)
-            model_weights = path.join(tmpdir, _MODEL_WEIGHTS)
-            conf = OmegaConf.load(config_yaml)
-            instance = cls.load_from_checkpoint(model_weights, cfg=conf)
+            # Prepare filenames.
+            config_yaml_file = path.join(tmpdir, _MODEL_CONFIG_YAML)
+            model_weights_file = path.join(tmpdir, _MODEL_WEIGHTS)
+
+            # Load the configuration.
+            config = OmegaConf.load(config_yaml_file)
+            print(config)
+
+            # Instantiate the object.
+            instance = Serialization.from_config_dict(config)
+
+            # Load the weights.
+            state_dict = load(model_weights_file, map_location=lambda storage, loc: storage)
```

Comment:
Please use explicit `torch.load()` here to have semantic understanding of which load we are calling here.

Reply:
this is wrong - should not be using torch.load(). This PR is fixing tests - I'll work on save/restore logic

Reply:
I didn't wanted to user torch.load, as I didn't wanted to call `import torch`

Reply:
Anyway, @okuchaiev  please, go ahead. 

Just leave the serialization to me, I really need to change several things down there to make it work for graphs.